guides/v2.1/cloud/project/new-relic.md

```diff
@@ -50,79 +42,87 @@ We recommend adding at least one Admin user to your New Relic account to manage
 
 1.  Ask the new user to check their email for a New Relic notification with account information.
 
-
 See [Accounts](https://docs.newrelic.com/docs/accounts/accounts) in the New Relic documentation for information about user roles, managing user accounts, and changing account ownership.
 
+## Configure New Relic
 
+Pro accounts are preconfigured to use the New Relic service; however, Starter accounts must add the New Relic service to the `.magento.app.yaml` configuration file and assign the license key to an environment variable. You can apply the New Relic license key to a maximum of three active environments.
```

Comment:
So for Pro accounts, you don't need to configure anything. Do you need to manage license keys for Pro environments?

If not, it might be clearer to make a heading:  " Add New Relic extension to your Starter project, and make it a level 3 heading.  

Same with the add a license key to an environment, remove license key -- if the instructions apply only to Starter projects.  Or find another way to clarify that you only need to do these things for Starter plan projects.

Reply:
This is already done for **new** starter projects, but may not be the case in all Starter projects. This configuration step is more of a verification.

I will not make an h3 in this case because the key is not part of the yaml configuration. 

Any Cloud project can manipulate variables in a paas environment.

Reply:
Your comments here confuse me. I have a sentence in the license key instructions that says "For Starter accounts, you need to use the license key and set a variable in the environment." I'm not sure why that is unclear. In fact, even if you are a Pro customer, you can do these steps. You probably don't need to, which is why I list how to find and verify things.

What I can do, is add a little more about verifying. I will think about it more.

Reply:
The main clarification I was looking for is that for Pro environments, you don't need to do the configuration since it's already done for you.  Since you can configure New Relic for integration branches on Starter, then it's likely that you might need to change the configuration if your integration environments change.  But on Pro (if I understand correctly), you only have New Relic on Staging and Production, and those branches are stable and not likely to change. 

On Pro, the keys are set up for you during provisioning?